content/getting-started/kotlin.textile

```diff
@@ -0,0 +1,183 @@
+    key = "{{API_KEY}}"
+    clientId = "my-first-client"
+  })
+
+  realtimeClient.connection.once(ConnectionEvent.connected) {
+    println("Made my first connection!")
+  }
+}
+
+fun main() {
+  getStarted()
+}
+```
+
+You can monitor the lifecycle of clients' connections, but for now just log a message to the console to know that the connection attempt was successful. You'll see the message printed to your console, and you can also inspect the connection event in the dev console of your app.
+
+h2(#step-2). Step 2: Subscribe to a channel and publish a message
+
+Messages contain the data that a client is communicating, such as a short 'hello' from a colleague, or a financial update being broadcast to subscribers from a server. Ably uses channels to separate messages into different topics, so that clients only ever receive messages on the channels they are subscribed to.
+
+* Add the following lines to your @getStarted()@ function to create a channel instance and register a listener to subscribe to the channel. Then run it:
+
+```[kotlin]
+val channel = ably.channels.get("my-first-channel")
+
+channel.subscribe { message ->
+  println("Received message: ${message.data}")
+}
+```
+
+* Use the Ably CLI to publish a message to your first channel. The message will be received by the client you've subscribed to the channel, and be logged to the console.
+
+```[sh]
+ably channels publish my-first-channel 'Hello!'
+```
+
+* In a new terminal tab, subscribe to the same channel using the CLI:
+
+```[sh]
+ably channels subscribe my-first-channel
+```
+
+Publish another message using the CLI and you will see that it's received instantly by the client you have running locally, as well as the subscribed terminal instance.
+
+h2(#step-3). Step 3: Join the presence set
+
+Presence enables clients to be aware of one another if they are present on the same channel. You can then show clients who else is online, provide a custom status update for each, and notify the channel when someone goes offline.
+
+* Add the following lines to your @getStarted()@ function to subscribe to, and join, the presence set of the channel. Then run it:
+
+```[kotlin]
+channel.presence.subscribe { presenceMessage ->
+    println("Event type: ${presenceMessage.action} from ${presenceMessage.clientId} with the data ${presenceMessage.data}")
+}
+
+channel.presence.enter("I'm here!", null)
+```
+
+* In the "dev console":https://ably.com/accounts/any/apps/any/console of your first app, attach to @my-first-channel@. Enter a @clientId@, such as @my-dev-console@, and then join the presence set of the channel. You'll see that @my-first-client@ is already present in the channel. In the console of your IDE you'll see that an event was received when the dev console client joined the channel.
+
+* You can have another client join the presence set using the Ably CLI: @ably channels presence enter my-first-channel --client-id "my-cli" --data '{"status":"online"}'@.
```

Comment:
Can we split the command out into a fenced block to improve the readability? Also change the status message:

`ably channels presence enter my-first-channel --client-id "my-cli" --data '{"status":"learning about Ably!"}'.`